uk.co.compucorp.civicrm.tasksassignments/views/contact/tasks.html

```diff
@@ -11,14 +11,17 @@
             {{settings.copy.button.assignmentAdd || 'Add Workflow'}}
           </button>
         </div>
-        <div class="col-xs-12 col-sm-6 col-lg-4" ng-if="settings.extEnabled.assignments">
+        <div class="col-xs-11 col-sm-5 col-lg-3" ng-if="settings.extEnabled.assignments">
           <ui-select theme="civihr-ui-select" prevent-animations multiple ng-model="list.filterParams.assignmentType">
             <ui-select-match prevent-animations class="ui-select-match" placeholder="Select workflow...">{{$item.title}}</ui-select-match>
             <ui-select-choices prevent-animations class="ui-select-choices" repeat="type.id as type in cache.assignmentType.arr">
               <div ng-bind-html="type.title | highlight: $select.search"></div>
             </ui-select-choices>
           </ui-select>
         </div>
+        <div class="col-xs-1 col-sm-1 col-lg-1">
```

Comment:
Please try to add a class `control-label` to align the wrench like here:

![image](https://user-images.githubusercontent.com/3973243/39815596-692c917e-5390-11e8-8188-4098e09546ef.png)

Also, I don't think you need col-sm-1 and col-lg-1.

```html
<div class="col-xs-1 control-label">
```